crypt.go

```diff
@@ -725,6 +740,24 @@ func InitializeLUKS2Container(devicePath, label string, key []byte, options *Ini
 				"--luks2-keyslots-size", fmt.Sprintf("%dk", options.KeyslotsAreaKiBSize))
 		}
 	}
+	if pbkdfOptions != nil {
+		if pbkdfOptions.PbkdfMmemory != 0 {
+			args = append(args,
+				"--pbkdf-memory", fmt.Sprintf("%d", pbkdfOptions.PbkdfMmemory))
+		}
+		if pbkdfOptions.PkdfForceIterations != 0 {
+			args = append(args,
+				"--pbkdf-force-iterations", fmt.Sprintf("%d", pbkdfOptions.PkdfForceIterations))
+		} else {
+			args = append(args, "--iter-time", "100")
+		}
+		if pbkdfOptions.PbkdfParallel != 0 {
```

Comment:
I'm not entirely sure why this is not under the if/else ?  I let @chrisccoulson comment/double check on this